nihms-native-assembler/src/test/java/org/dataconservancy/nihms/assembler/nihmsnative/NihmsAssemblerIT.java

```diff
@@ -159,14 +159,16 @@ public void testPackageMetadata() throws Exception {
         }).collect(Collectors.toList());
 
         // Assert that each person present in the Submission is present in the metadata
-        assertEquals(submission.getMetadata().getPersons().size(), personElements.size());
+        assertEquals(1, personElements.size());
         submission.getMetadata().getPersons().forEach(p -> {
-            assertTrue(asPersons.stream().anyMatch(candidate ->
-                    candidate.getFirstName().equals(p.getFirstName()) &&
-                    candidate.getLastName().equals(p.getLastName()) &&
-                    candidate.isAuthor() == p.isAuthor() &&
-                    candidate.isPi() == p.isPi() &&
-                    candidate.isCorrespondingPi() == p.isCorrespondingPi()));
+            if (p.isCorrespondingPi()) {
```

Comment:
Based on this PR it looks like:
1. None or One `Person` can be associated with the NIHMS submission
2. If  there is a `Person` associated with the NIHMS submission, then that Person _must_ be a corresponding PI.

If that's the case, then the assertion in this test needs to be stronger:
instead of `if (p.isCorrespondingPi())` the test should probably _assert_ that the `Person` is a corresponding PI immediately, rather than surrounding the following assertions in the `if ...` conditional.